### PR TITLE
Feat/63 text changes in view

### PR DESCRIPTION
**Pull request by sergiomarrocoli**
Some changes to what is displayed, and removes db fields that we aren't using any more.

Testing:
- reset db and check `rake import_commitments_and_migrate_data` still runs correctly
- on commitments index, check 'Duration' field displays correctly. only csv imports will have this, so the first 21 commitments 
- check show page text changes:
-- Expected duration once fully implemented  > Expected duration of commitment
-- Year when the commitment was fully implemented >  First year of commitment actions
-- check Proposed area and Review method have been removed

### CODE REVIEW

File: app/views/commitments/show.html.erb [hunk 1]
```diff
@@ -33,21 +33,17 @@
...
-          <span class="list__subtitle">Proposed area:</span>
-          <span class="list__text"><%= @commitment.proposed_area_ha %> hectares</span>
-        </li>
         <li class="list__li">
           <span class="list__subtitle">Year when the commitment was made:</span>
           <span class="list__text"><%= @commitment.committed_year %></span>
         </li>
         <li class="list__li">
-          <span class="list__subtitle">Year when the commitment was fully implemented:</span>
+          <span class="list__subtitle">First year of commitment actions:</span>
```
Comment (yndajas) [inline app/views/commitments/show.html.erb:41]: Not your code, but could implement translations in this file while editing?

File: db/migrate/20220411085523_remove_duration_and_review_method_from_commitments.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class RemoveDurationAndReviewMethodFromCommitments < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :commitments, :duration, :string
+    remove_column :commitments, :review_method, :string
+    remove_column :commitments, :proposed_area_ha, :string
```
Comment (yndajas) [inline db/migrate/20220411085523_remove_duration_and_review_method_from_commitments.rb:5]: If you're removing these columns, do we need the `ignore_column`s on the model?

**Review by yndajas — CHANGES_REQUESTED** [review]
Looks good. Just a couple small comments

File: db/migrate/20220411085523_remove_duration_and_review_method_from_commitments.rb (hunk 2 above)
Comment (sergiomarrocoli) [reply db/migrate/20220411085523_remove_duration_and_review_method_from_commitments.rb:5]: ignore_columns in the gem means if it sees a column in the csv it doesn't try to import it, so can either change the csv, or add this

Comment (yndajas) [reply db/migrate/20220411085523_remove_duration_and_review_method_from_commitments.rb:5]: Ah I see!

File: app/views/commitments/show.html.erb [hunk 3]
```diff
@@ -21,39 +21,39 @@
...
 
     <div class="list--commitment-info">
-      <h2 class="list__title"><span class="green">Stage:</span> <%= @commitment.stage %></h2>
+      <h2 class="list__title"><span class="green"><%= I18n.t('commitments.show.stage') %>:</span> <%= @commitment.stage %></h2>
 
       <ul class="list__ul">
         <li class="list__li">
-          <span class="list__subtitle">Actors:</span>
-          <span class="list__text"><%= @commitment.manager&.name %></span>
+          <span class="list__subtitle"><%= I18n.t('commitments.show.actors') %></span>
```
Comment (yndajas) [inline app/views/commitments/show.html.erb:28]: Colon retained (outside translation) on other lines, but not here

**Review by yndajas — CHANGES_REQUESTED** [review]
Just one typo, then it's good to go

**Pull request merged**